contracts/Donations.sol

```diff
@@ -130,8 +132,6 @@ contract Donations is ERC721, AccessControl {
         uint256 _metadataId = metadataId;
 
         for (uint256 i = 0; i < length; ++i) {
-            ++_metadataId;
```

Comment:
This was before on purpose, because there was some feedback to make this start on 1 like the claimers and depositors.

Reply:
forgot to comment but this has been fixed again